lr_face/data.py

```diff
@@ -154,6 +154,24 @@ def same_identity(self) -> bool:
         """
         return self.first.identity == self.second.identity
 
+    # Experts' log LLR estimation (optional).
+    @property
+    def expertsLLR(self) -> [np.array]:
+        # read in log LR from experts file
+        folder = os.path.dirname(self.first.path)
+        experts_path = os.path.join(folder, "Experts_LLR.csv")
+        if os.path.isfile(experts_path):
+            with open(os.path.join(experts_path)) as exprt:
+                reader = pd.read_csv(exprt)
+                line = reader.loc[reader['id'] == self.first.meta['idx']].to_numpy(dtype='float16')
+                experts = line[0, 1:]
+                return experts
+        else:
+            if 'Enfsi' in self.first.source:
```

Comment:
maybe check this in the beginning? Otherwise you keep trying to find the file for images in other datasets. And what about if the second image is from a non-enfsi source?